src/renderer/src/hooks/use-sso.ts

```
@@ -0,0 +1,26 @@
+import { useEffect } from "react";
+import { useSSOStore } from "../store/sso-store";
+import { useAWSStore } from "@renderer/store/aws-store";
+
+export const useSSO = () => {
+  const { awsConfig } = useAWSStore();
+  const { isAuthenticated, accessToken, expiresAt, startSSO } = useSSOStore();
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: awsConfig[0]

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)